scripts/android/build_decred.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+. ./config.sh
+CW_DECRED_DIR=${WORKDIR}/cake_wallet/cw_decred
+LIBWALLET_PATH="${WORKDIR}/decred/libwallet"
+LIBWALLET_URL="https://github.com/itswisdomagain/libwallet.git"
+
+if [ -e $LIBWALLET_PATH ]; then
+       rm -fr $LIBWALLET_PATH
+fi
+mkdir -p $LIBWALLET_PATH
+git clone $LIBWALLET_URL $LIBWALLET_PATH --branch cgo
```

Comment:
I think this should point at the specific commit that this pr should work on. That is possible right? e1b9f6043359d191cfd2fbfaa345396a009d0946

In case we make a breaking change, which I think is fine at this point.